src/main/java/neko/Parser.java

```diff
@@ -0,0 +1,56 @@
+package neko;
+import neko.task.*;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class Parser {
+    private static final DateTimeFormatter parseFormatter =
+            DateTimeFormatter.ofPattern("eee, d MMM uuuu h:mma");
+    private static final DateTimeFormatter dateFormatter =
+            DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm");
+
+    public static Task parseTask(String str) {
+        char c = str.charAt(0);
```

Comment:
Perhaps a more intuitive variable name for c here? taskType might make the code easier to read at a glance, especially for people unfamiliar with this part of the code. What do you think?